mlrun/api/api/endpoints/runs.py

```diff
@@ -283,3 +283,43 @@ async def delete_runs(
         days_ago,
     )
     return {}
+
+
+@router.put(
+    "/projects/{project}/runs/{uid}/notifications",
+    status_code=HTTPStatus.ACCEPTED.value,
```

Comment:
Why return `ACCEPTED`? The modification happens immediately, so why not return 200?